spark-statsd-reporter/src/main/java/org/apache/spark/metrics/sink/statsd/StatsdReporterFactory.java

```diff
@@ -0,0 +1,66 @@
+package org.apache.spark.metrics.sink.statsd;
+
+import com.codahale.metrics.MetricRegistry;
+
+import io.dropwizard.metrics.BaseReporterFactory;
+
+public class StatsdReporterFactory extends BaseReporterFactory {
+	
+    private String host = "127.0.0.1";
+    private int port = 8125;
+    private String prefix = "";
+    private String reporterName = "spark-statsd-reporter";
+    private MetricFormatter metricFormatter = null;
+
+    public String getHost() {
```

Comment:
this method is not used anywhere, let's remove it

Reply:
Removed It.